code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -217,12 +217,12 @@ can cause issues with ammo types getting mixed up during the burst.
 
 
 /obj/item/weapon/gun/shotgun/combat/set_gun_config_values()
-	fire_delay = config.mhigh_fire_delay*2
-	accuracy_mult = config.base_hit_accuracy_mult
-	accuracy_mult_unwielded = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult - config.hmed_hit_accuracy_mult
+	fire_delay = config.high_fire_delay * 3 //one shot every 1.5 seconds.
+	accuracy_mult = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult
+	accuracy_mult_unwielded = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult - config.max_hit_accuracy_mult //you need to wield this gun for any kind of accuracy
 	scatter = config.med_scatter_value
-	scatter_unwielded = config.max_scatter_value
-	damage_mult = config.base_hit_damage_mult - config.low_hit_damage_mult
+	scatter_unwielded = config.max_scatter_value * 1.25 //50% scatter chance unwielded.
```

Comment:
You know, the config values are there for a reason. So we don't have magical numbers.